controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -104,6 +115,11 @@ def run_match(supervisor: Supervisor) -> None:
     print("Match start")
     print("===========")
 
+    # First signal the robot controllers that they're able to start ...
+    for _, robot in get_robots(supervisor):
+        robot.getField('customData').setSFString('start')
```

Comment:
Thoughts on making `"start"` and `"ready"` some form of constant?

Reply:
Maybe, though I expect that the import shenanigans we'd need to be able to share them would be more trouble than it's worth.

Reply:
Yeah that's true. There's still some benefit to defining them in a constant place. We can use comments to ensure they're updated consistently in future.

Reply:
c475411